Formula/qgis3.rb

```diff
@@ -312,8 +314,10 @@ def install
     # install db plugins to local qt plugins prefix
     if build.with? "qspatialite"
       mkdir lib_qt/"plugins/sqldrivers"
-      inreplace "src/providers/spatialite/CMakeLists.txt",
-                "${QT_PLUGINS_DIR}/sqldrivers", lib_qt/"plugins/sqldrivers".to_s
+      # inreplace "src/providers/spatialite/CMakeLists.txt",
```

Comment:
In the future, please don't comment out old code, it makes the formula harder to read, and we can get the diffs via the git log, if we need to.

Reply:
You are right! sometimes I leave them for me :joy: